src/main/java/net/dv8tion/jda/core/EmbedBuilder.java

```diff
@@ -173,6 +173,26 @@ public EmbedBuilder setTitle(String title, String url)
     }
 
     /**
+     * The text of the embed title, set using {@link EmbedBuilder#setTitle(String, String)}.
+     *
+     * @return Possibly-null String containing the title of the embedded resource.
+     */
+    public String getTitle()
+    {
+        return title;
+    }
+
+    /**
+     * The URL of the embed title, set using {@link EmbedBuilder#setTitle(String, String)}.
+     *
+     * @return URL of the title of the embed or {@code null} if it is not set.
+     */
+    public String getUrl()
```

Comment:
Here you should mention that the url can only be non-null if `getTitle` isn't null either.